app/views/teachers/CreateTeacherAccountView.coffee

```diff
@@ -61,7 +61,7 @@ module.exports = class CreateTeacherAccountView extends RootView
     @listenTo @state, 'change:error', -> @renderSelectors('.error-area')
     @listenTo @state, 'change:showUsaStateDropdown', -> @renderSelectors('.state')
     @listenTo @state, 'change:stateValue', -> @renderSelectors('.state')
-    loadSegment() unless @segmentLoaded
+#    loadSegment() unless @segmentLoaded
```

Comment:
Comments required or handled in one of the later PRs?

Reply:
This isn't needed anymore and has been handled here, should have removed this line but forgot.  Removed!